### PR TITLE
Rest of the memkind documentation

### PR DESCRIPTION
**Pull request by szadam**
I will categorize the documentation later ;)

- - -
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)

### CODE REVIEW

**Comment by szadam** [comment]
Preview:
https://szadam.github.io/memkind/

**Review by maciekpac — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: 0 of 20 files reviewed, 1 unresolved discussion (waiting on @bratpiorka, @PatKamin, and @szadam)

___
*[`-- commits` line 2 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9Xz42DGUjllH_JRi1):*
I think this commit message is not grammatically correct.

The world *existing* needs to be followed by the object, e.g.

Add the rest of the documentation and **introduce** small amendments to the existing **docs**

or something like that

**Review by lukaszstolarczuk — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: 0 of 20 files reviewed, 2 unresolved discussions (waiting on @bratpiorka, @PatKamin, and @szadam)

___
*[`content/memkind/manpages/autohbw.7.md` line 10 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9jxD07gQ-tzjEH2lD:-NA9jxD07gQ-tzjEH2lE:bi82w1h) ([raw file](https://github.com/pmem/pmem.github.io/blob/5feb0b81cd1bf474354fa699de6dfcc1586f3c68/content/memkind/manpages/autohbw.7.md#L10)):*
> ```Markdown
> title: AUTOHBW
> section: 7
> ---
> ```

shouldn't these new manpages be linked on the /memkind page? of course, unless you want them to be only accessible from other links (like from another manpages)

**Review by PatKamin — COMMENTED** [review]
> Reviewed 6 of 20 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: 6 of 20 files reviewed, 2 unresolved discussions (waiting on @bratpiorka, @lukaszstolarczuk, and @szadam)

___
*[`content/memkind/manpages/autohbw.7.md` line 10 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9jxD07gQ-tzjEH2lD:-NA9u4xbCTKHFKVA2QPq:b4rug6) ([raw file](https://github.com/pmem/pmem.github.io/blob/5feb0b81cd1bf474354fa699de6dfcc1586f3c68/content/memkind/manpages/autohbw.7.md#L10)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

shouldn't these new manpages be linked on the /memkind page? of course, unless you want them to be only accessible from other links (like from another manpages)
</blockquote></details>

That's right, all of them should be accessible from the /memkind page.

**Review by szadam — COMMENTED** [review]
> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: 8 of 22 files reviewed, 2 unresolved discussions (waiting on @bratpiorka, @lukaszstolarczuk, and @maciekpac)

___
*[`-- commits` line 2 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9Xz42DGUjllH_JRi1):*
<details><summary><i>Previously, maciekpac wrote…</i></summary><blockquote>

I think this commit message is not grammatically correct.

The world *existing* needs to be followed by the object, e.g.

Add the rest of the documentation and **introduce** small amendments to the existing **docs**

or something like that

</blockquote></details>

You're right. I have changed it.
___
*[`content/memkind/manpages/autohbw.7.md` line 10 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9jxD07gQ-tzjEH2lD:-NAJcfjK6tyOo5nXOjCF:b597q5g) ([raw file](https://github.com/pmem/pmem.github.io/blob/5feb0b81cd1bf474354fa699de6dfcc1586f3c68/content/memkind/manpages/autohbw.7.md#L10)):*
<details><summary><i>Previously, PatKamin (Patryk Kaminski) wrote…</i></summary><blockquote>

That's right, all of them should be accessible from the /memkind page.
</blockquote></details>

Yes, of course. I have added and categorized it.

**Review by lukaszstolarczuk — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: 8 of 22 files reviewed, 5 unresolved discussions (waiting on @bratpiorka, @maciekpac, and @szadam)

___
*[`-- commits` line 2 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9Xz42DGUjllH_JRi1):*
<details><summary><i>Previously, szadam (Adam Szopiński) wrote…</i></summary><blockquote>

You're right. I have changed it.
</blockquote></details>

now, this commit msg is rather long and very verbose. I believe you can completely re-write it, IMHO, it's too wordy, too descriptive; or just consider splitting the commit msg into two lines
___
*[`content/memkind/manpages/autohbw.7.md` line 15 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAJmfiX3LB-JoQxe3Bc:-NAJmfiX3LB-JoQxe3Bd:b-s76s8j) ([raw file](https://github.com/pmem/pmem.github.io/blob/122a12ad38b4aa8b63e1ae095744407962388d09/content/memkind/manpages/autohbw.7.md#L15)):*
> ```Markdown
> [comment]: <> (Copyright 2014-2022, Intel Corporation)
> 
> [comment]: <> (autohbw.7 -- man page for fixedallocator)
> ```

for ...?
___
*[`content/memkind/manpages/memkind-hbw-nodes.1.md` line 15 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAJmqSfBxtBjYCWqEN-:-NAJmqSfBxtBjYCWqEN0:b-s76s8j) ([raw file](https://github.com/pmem/pmem.github.io/blob/122a12ad38b4aa8b63e1ae095744407962388d09/content/memkind/manpages/memkind-hbw-nodes.1.md#L15)):*
> ```Markdown
> [comment]: <> (Copyright 2016-2022, Intel Corporation)
> 
> [comment]: <> (memkind-hbw-nodes.1 -- man page for memkind-auto-dax-kmem-nodes)
> ```

for ...?
___
*[`data/en/memkind.yml` line 24 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAJnLdpA9BEPmaMlXsC:-NAJnLdpA9BEPmaMlXsD:b-ptqvoe) ([raw file](https://github.com/pmem/pmem.github.io/blob/122a12ad38b4aa8b63e1ae095744407962388d09/data/en/memkind.yml#L24)):*
> ```YAML
>           content: "<a href=\"./manpages/memkind.3.html\">memkind(3) API</a>"
>         list_item02:
>           content: "<a href=\"./manpages/memkind_arena.3.html\">memkind_arena(3)</a>"
> ```

maybe you consider describing these manpages a little - some are commands, some are libraries, some include APIs, ... 

not necessarily it has to be done in this PR, but it'd be nice to have some short description for all/some of these manpages
___
*[`data/en/memkind.yml` line 60 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAJnlGG1yAfqbMQMeFc:-NAJnlGHBXmfnZowJIOF:b-i08lka) ([raw file](https://github.com/pmem/pmem.github.io/blob/122a12ad38b4aa8b63e1ae095744407962388d09/data/en/memkind.yml#L60)):*
> ```YAML
>           content: "<a href=\"./manpages/fixedallocator.3.html\">fixedallocator(3) API</a>"
>         list_item04:
>           content: "<a href=\"./manpages/memkindallocator.3.html\">memkind allocator(3) API</a>"
> ```

here you wrote ".. allocator" with space and above you did not - perhaps unify, pls...?

**Review by szadam — COMMENTED** [review]
> Reviewed 20 of 20 files at r1, 3 of 3 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: all files reviewed, 4 unresolved discussions (waiting on @bratpiorka, @lukaszstolarczuk, and @maciekpac)

___
*[`-- commits` line 2 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9Xz42DGUjllH_JRi1):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

now, this commit msg is rather long and very verbose. I believe you can completely re-write it, IMHO, it's too wordy, too descriptive; or just consider splitting the commit msg into two lines
</blockquote></details>

I propose something simple like this
___
*[`content/memkind/manpages/autohbw.7.md` line 15 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAJmfiX3LB-JoQxe3Bc:-NAiG49k54Sc7rj2PPYC:b-yigp24) ([raw file](https://github.com/pmem/pmem.github.io/blob/122a12ad38b4aa8b63e1ae095744407962388d09/content/memkind/manpages/autohbw.7.md#L15)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

for ...?
</blockquote></details>

corrected
___
*[`content/memkind/manpages/memkind-hbw-nodes.1.md` line 15 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAJmqSfBxtBjYCWqEN-:-NAiG9lY1WuL899KIgaA:bapfhvs) ([raw file](https://github.com/pmem/pmem.github.io/blob/122a12ad38b4aa8b63e1ae095744407962388d09/content/memkind/manpages/memkind-hbw-nodes.1.md#L15)):*
> corrected
___
*[`data/en/memkind.yml` line 24 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAJnLdpA9BEPmaMlXsC:-NAiGD0zE7fU-h1Vgcq0:b7y202q) ([raw file](https://github.com/pmem/pmem.github.io/blob/122a12ad38b4aa8b63e1ae095744407962388d09/data/en/memkind.yml#L24)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

maybe you consider describing these manpages a little - some are commands, some are libraries, some include APIs, ... 

not necessarily it has to be done in this PR, but it'd be nice to have some short description for all/some of these manpages
</blockquote></details>

I think, that this could be added in other PR

**Review by lukaszstolarczuk — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: all files reviewed, 2 unresolved discussions (waiting on @bratpiorka, @maciekpac, and @szadam)

___
*[`-- commits` line 2 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9Xz42DGUjllH_JRi1):*
<details><summary><i>Previously, szadam (Adam Szopiński) wrote…</i></summary><blockquote>

I propose something simple like this
</blockquote></details>

this repo holds data for various sub-projects - it'd be perfect to mention it's for memkind :wink: 
___
*[`content/memkind/manpages/memkind-hbw-nodes.1.md` line 15 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAJmqSfBxtBjYCWqEN-:-NAiLIAQ-e8GHK3cUJI6:b-alolis) ([raw file](https://github.com/pmem/pmem.github.io/blob/122a12ad38b4aa8b63e1ae095744407962388d09/content/memkind/manpages/memkind-hbw-nodes.1.md#L15)):*
<details><summary><i>Previously, szadam (Adam Szopiński) wrote…</i></summary><blockquote>

> corrected
</blockquote></details>

nope...?

**Review by PatKamin — COMMENTED** [review]
> Reviewed 1 of 3 files at r3.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: all files reviewed, 3 unresolved discussions (waiting on @bratpiorka, @maciekpac, and @szadam)

___
*[`data/en/memkind.yml` line 49 at r3](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAmojKH3Ox7sheVAqZ6:-NAmojKI1UvNDHut65kb:b-tdttad) ([raw file](https://github.com/pmem/pmem.github.io/blob/69e49cb4f2522dcdfeaef4c050e6f83c3c0e3ef1/data/en/memkind.yml#L49)):*
> ```YAML
>           content: "<a href=\"./manpages/memkind-hbw-nodes.1.html\">memkind-hbw-nodes(1)</a>"
>         list_item05:
>           content: "<a href=\"./manpages/memkind_hbw.3.html\">memkind_hbw(3)</a>"
> ```

nit: Least relevant last:
1. hbwmalloc(3)
2. memkind_hbw(3)
3. autohbw(7)
4. memkind-hbw-nodes(1)



_Code quote:_
```YAML
        list_item01:
          content: "<a href=\"./manpages/hbwmalloc.3.html\">hbwmalloc(3)</a>"
        list_item03:
          content: "<a href=\"./manpages/autohbw.7.html\">autohbw(7)</a>"
        list_item04:
          content: "<a href=\"./manpages/memkind-hbw-nodes.1.html\">memkind-hbw-nodes(1)</a>"
        list_item05:
          content: "<a href=\"./manpages/memkind_hbw.3.html\">memkind_hbw(3)</a>"
```

**Review by szadam — COMMENTED** [review]
> Reviewed 2 of 2 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: all files reviewed, 2 unresolved discussions (waiting on @bratpiorka, @lukaszstolarczuk, and @maciekpac)

___
*[`-- commits` line 2 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9Xz42DGUjllH_JRi1):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

this repo holds data for various sub-projects - it'd be perfect to mention it's for memkind :wink: 
</blockquote></details>

okey ;)
___
*[`content/memkind/manpages/memkind-hbw-nodes.1.md` line 15 at r2](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NAJmqSfBxtBjYCWqEN-:-NAmwQNj9vhlkzhsz40C:bijdbck) ([raw file](https://github.com/pmem/pmem.github.io/blob/122a12ad38b4aa8b63e1ae095744407962388d09/content/memkind/manpages/memkind-hbw-nodes.1.md#L15)):*
<details><summary><i>Previously, lukaszstolarczuk (Łukasz Stolarczuk) wrote…</i></summary><blockquote>

nope...?
</blockquote></details>

okey, now it is corrected ;D

**Review by szadam — COMMENTED** [review]
> Reviewed 1 of 1 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297)* status: all files reviewed, 2 unresolved discussions (waiting on @bratpiorka, @lukaszstolarczuk, and @maciekpac)

**Review by lukaszstolarczuk — COMMENTED** [review]
:lgtm: 

> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297#-:-NAmyy0Y5ZvZ5RlBihNG:bg7igkv)* status: all files reviewed, 1 unresolved discussion (waiting on @bratpiorka and @maciekpac)

___
*[`-- commits` line 2 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/297#-NA9Xz42DGUjllH_JRi1):*
<details><summary><i>Previously, szadam (Adam Szopiński) wrote…</i></summary><blockquote>

okey ;)
</blockquote></details>

thx!

**Review by pbalcer — APPROVED** [review]
:lgtm:

> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/297#-:-NAnSL2K9fokbrA3mqHI:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @bratpiorka and @maciekpac)
